x-pack/plugins/security_solution/public/common/components/formatted_date/index.tsx

```diff
@@ -97,16 +97,18 @@ interface FormattedDateProps {
   className?: string;
   fieldName: string;
   value?: string | number | null;
+  dateFormat?: string;
```

Comment:
Can we add the unit test that this setting will change the behavior properly?

Reply:
yup! added some.